flink-runtime/src/test/java/org/apache/flink/runtime/testutils/PseudoRandomValueSelectorTest.java

```diff
@@ -34,25 +34,20 @@
 import static org.apache.flink.configuration.CheckpointingOptions.SAVEPOINT_DIRECTORY;
 import static org.apache.flink.configuration.JobManagerOptions.JOB_MANAGER_HEAP_MEMORY_MB;
 import static org.apache.flink.configuration.TaskManagerOptions.CPU_CORES;
-import static org.hamcrest.MatcherAssert.assertThat;
-import static org.hamcrest.Matchers.greaterThan;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertNull;
-import static org.junit.Assert.assertTrue;
-import static org.junit.Assume.assumeFalse;
+import static org.assertj.core.api.Assertions.assertThat;
 import static org.junit.Assume.assumeNoException;
 import static org.junit.Assume.assumeNotNull;
+import static org.junit.jupiter.api.Assumptions.assumeFalse;
 
 /** Tests {@link PseudoRandomValueSelector}. */
-public class PseudoRandomValueSelectorTest extends TestLogger {
+class PseudoRandomValueSelectorTest extends TestLogger {
```

Comment:
We should replace usage of TestLogger with TestLoggerExtension

Reply:
Thanks @X-czh  for the review, I will remove the `TestLogger` later. And this PR isn't to ready, I mark it to a draft PR.

I will let you know after it's ready.